packages/shared/src/components/sidebar/SquadSection.tsx

```diff
@@ -45,6 +45,9 @@ export function SquadSection(props: SectionCommonProps): ReactElement {
         ),
       title: name,
       path: permalink,
+      className: {
+        text: '!block',
```

Comment:
What is the default here that we need the override? This feels like it might add a edge case somehow.

Reply:
`flex`. Couldn't get it to add the ellipsis with flex. Works with everything else but flex and inline flex.

This just adds `block` span that wraps around the squad name. Shouldn't affect anything else.

Reply:
Hmm since that already has `flex` and alignments it might though.
Either those become redundant or we removed vertical centering for instance?

I think we should either fully remove the flex on that last child or see if we can fix it with the [min-width fix](https://css-tricks.com/flexbox-truncated-text/) described here?
Let me know what you think.

Reply:
I tried the `min-width: 0` hack on all of the items in that hierarchy, but none seems to do the trick.

The flex that is being overridden in this case should not have any use-case anyway, as it's just a single span wrapping text, no html inside it. 
![image](https://github.com/dailydotdev/apps/assets/1681525/9c89adbf-b194-4870-9bb7-e844744a2324)
![image](https://github.com/dailydotdev/apps/assets/1681525/eb246488-9ef3-48be-981f-6c430d8da3e1)

Removing `items-center` on that `span` from this span does not seem to have any effect either, as the parent `a` both has `flex` and `items-center`.

Reply:
I thought about removing flex entirely from the `span` inside the `ItemInner` entirely, but didn't want to do that in case the `flex` might have an effect the other places it's being used.

Reply:
Ah yes so let;s then remove the flex related classes as they might confuse people right (if not used now)

Reply:
Removed them, now I also don't need to add the block class

Reply:
Just to be sure, I will assign a preview to the pr and go through it to verify nothing is broken.

Reply:
Checked on desktop and mobile, nothing seems to have been broken.